packages/ui-components/src/lib/storesGeneric/cachedWritableStore.ts

```
@@ -0,0 +1,131 @@
+ * Creates a writable store that can hold an optional number value and persists to localStorage.
+ *
+ * @param {string} key - The localStorage key to use for persistence
+ * @param {number | undefined} defaultValue - The default value if nothing is found in localStorage
+ * @returns A writable store that persists to localStorage and can hold an optional number
+ */
+export const cachedWritableIntOptional = (key: string, defaultValue = undefined) =>
+	cachedWritableOptionalStore<number>(
+		key,
+		defaultValue,
+		(v) => v.toString(),
+		(v) => parseInt(v)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)